openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+            "reset" : "リセット",
+            "close" : "閉じる",
+            "clear": "クリア",
+            "logout" : "ログアウト",
+            "register" : "登録",
+            "home" : "ホーム",
+            "start" : "開始",
+            "name" : "名前",
+            "continue" : "継続",
+            "next" : "次へ",
+            "back" : "戻る",
+            "delete" : "削除",
+            "details" : "詳細",
+            "pleaseSelect" : "選択してください",
+            "select" : "選択",
+            "addUser" : "ユーザーの追加",
+            "decision" : "決定",
+            "accept" : "許可",
+            "reject" : "却下",
+            "createdAt" : "に作成されました",
+            "createNew": "新規作成",
+            "search" : "検索",
+            "submit" : "サブミット",
+            "actions" : "アクション",
+            "complete" : "完了",
+            "revoke" : "取り消し",
+            "done" : "終了",
+            "ok" : "OK",
+            "moveUp" : "上に移動",
+            "moveDown" : "下に移動",
+            "standard": "標準",
+            "advanced": "詳細設定",
+            "revert": "元に戻す",
+            "saveChanges": "変更の保存",
+            "edit": "編集",
+            "key":"キー",
+            "value": "値",
+            "addKey" : "キーの追加",
+            "addValue" : "値の追加",
+            "settings" : "設定",
+            "working" : "処理中...",
+            "more" : "more...",
+            "empty" : "空",
+            "help" : "ヘルプ",
+            "change": "変更",
+            "upload": "アップロード",
+            "download": "ダウンロード",
+            "validate": "検証",
+            "filter": "フィルタ...",
+            "pass" : "成功",
+            "fail" : "失敗",
+            "success" : "成功",
+            "allow": "許可",
+            "deny": "拒否",
+            "notApplicable": "n/a",
+            "enable": "有効",
+            "disable": "無効",
+            "enabled": "有効",
+            "disabled": "無効",
+            "activate": "アクティブ",
+            "deactivate": "非アクティブ",
+            "active": "アクティブ",
+            "inactive": "非アクティブ",
+            "copyright" : "Copyright &copy; 2010-2016 ForgeRock AS. All rights reserved.",
+            "build": "ビルド",
+            "sessionExpired" : "セッションが有効期限切れ",
+            "enableCookies" : "ログインするためには、cookie を有効にする必要があります",
+            "deleteSelected" : "選択したものを削除します",
+            "startTypingToSearch": "検索する入力を開始",
+            "passwordPlaceholder": "••••••••••••",
+            "redirecting": "リダイレクトします...",
+            "propertyName": "プロパティ名",
+            "propertyValue": "プロパティ値",
+            "addName": "名の追加",
+            "addValue": "値の追加",
+            "validation" : {
+                "errorsNotSaved" : "フォームに検証エラーが存在します。変更は保存できません。",
+                "VALID_PHONE_FORMAT": "無効な文字が含まれています。",
+                "VALID_NAME_FORMAT": "無効な文字が含まれています。",
+                "VALID_EMAIL_ADDRESS_FORMAT": "電子メールアドレスが無効です。",
+                "AT_LEAST_X_CAPITAL_LETTERS" : "最低 __numCaps__ capital 文字以上は必要です。",
+                "AT_LEAST_X_NUMBERS" : "少なくとも __numNums__ 個以上の数字が必要です。",
```

Please unify the words(`少なくとも` or `最低`).

---

There were other similar problems.
However, because each group is unified, I will change only the place pointed out.
```suggestion
                "AT_LEAST_X_NUMBERS" : "最低 __numNums__ 個以上の数字が必要です。",
```